spec/system/claims/support/schools/claims/view_a_schools_claims_spec.rb

```diff
@@ -0,0 +1,36 @@
+require "rails_helper"
+
+RSpec.describe "View a schools claims", type: :system, service: :claims do
+  let!(:school) { create(:school, :claims).becomes(Claims::School) }
+  let!(:colin) { create(:persona, :colin, service: "claims") }
+  let!(:claim) { create(:claim, school_id: school.id) }
+
+  scenario "View a school's claims as a support user" do
+    given_i_sign_in_as_colin
+    when_i_visit_the_claims_support_school_claims_page
+    i_see_a_list_of_the_schools_claims
+  end
+
+  private
+
+  def given_i_sign_in_as_colin
+    and_i_visit_the_personas_page
+    and_i_click_sign_in_as("Colin")
+  end
+
+  def and_i_visit_the_personas_page
+    visit personas_path
+  end
+
+  def and_i_click_sign_in_as(persona_name)
+    click_on "Sign In as #{persona_name}"
+  end
+
+  def when_i_visit_the_claims_support_school_claims_page
+    visit claims_support_school_claims_path(school)
+  end
+
+  def i_see_a_list_of_the_schools_claims
+    expect(page).to have_content("Claims\nID #{claim.id} Submitted")
```

Comment:
Could you also add a test to show that a "Draft" claim shows as "Draft" and possibly add a random claim (_not associated with that school)_ to demonstrate that only school related claims are returned.